tests/utils.py

```diff
@@ -5,20 +5,26 @@
 
 Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
 """
+import uuid
 from collections import (
     namedtuple,
 )
 from datetime import (
     datetime,
 )
+from functools import (
```

Comment:
In this case, doesn't matter so much, but it's preferable to use `from cached_property import cached_property` as it's a better implementation (cache can be flushed at (instance, method) level, instead of the `functools` one, which can be flushed only at `method` level)